recipes-core/rhino/rhino_1.7r4.bb

```diff
@@ -46,5 +47,5 @@ do_install:append() {
 	install -m 0755 ${WORKDIR}/rhino-jsc ${D}${bindir}
 }
 
-SRC_URI[md5sum] = "ad67a3dff135e3a70f0c3528a2d6edf2"
-SRC_URI[sha256sum] = "9eb08f85bbe7c8e0b9eaffb1cf4984b31fb679f3c8a682acc3bb8ac20626c61e"
+SRC_URI[md5sum] = "1f893577269631703d31e4de9d5dc1f4"
+SRC_URI[sha256sum] = "860965fc611764745b3a4fc5bd4baac07356a9fedd2ce6642e7bb0bd7ef58d07"
```

Comment:
Why do you need a checksum here? Isn't the SRCREV sufficient?

Reply:
Forgot to remove that. Just pushed changes